### PR TITLE
Typos/editorial comments

### PR DESCRIPTION
**Pull request by pinamiranda**
Typos: 

+ Line 221: "display" -> "present"

+ Line 459 - 461: “The requirements […] are indented” → intended
+ Line 573: “enable the EUDI Wallet to work smooth” → smoothly

+ Line 760: why is the acronym for 'PID Issuance Interface' PPI instead of PII ?

+ Line 920: “the ng PID Providers” 

+ Line 1272: “As a of the registration' --> "As a consequence of the registration"

+ Lines 1366-1367: “ Relying Parties may try to request attributes from a Wallet Instance for which they have lawful grounds.” → “no lawful ground” ?

+ Line 1478: “to the Attestation Provide” -> Provider

### CODE REVIEW

File: docs/arf.md [hunk 1]
```diff
@@ -912,12 +912,12 @@ eID scheme.
...
 
--   the certification of the Wallet Solution by accredited and
+- the certification of the Wallet Solution by accredited and
     designated CAB(s),
 
--   ex-ante supervisory activities by Member State supervisory bodies,
+- ex-ante supervisory activities by Member State supervisory bodies,
 
--   both the body providing EUDI Wallets and the ng PID Providers have
+- both the body providing EUDI Wallets and the PID Providers have
```
Comment (skounis) [inline docs/arf.md:920]: I saw this fix in another PR so let's be careful when we merge so we avoid any conflicts

File: docs/arf.md [hunk 2]
```diff
@@ -1364,7 +1364,7 @@ trust in the EUDI Wallet ecosystem:
         or mDL attestation type.
 
     -   Relying Parties may try to request attributes from a Wallet
-        Instance for which they have lawful grounds.
+        Instance for which they have no lawful grounds.
```
Comment (skounis) [inline docs/arf.md:1367]: This is also part of another one PR, so let's pay attention when we merge

**Review by skounis — APPROVED** [review]

**Comment by peppelinux** [comment]
thank you @pinamiranda

**Review by digeorgi — APPROVED** [review]
We can proceed with the proposed changes

**Pull request merged**